src/js/scroll_panel.js

```diff
@@ -8,9 +8,11 @@
  *
  */
 
-import {css, log_perf, html, throttlePromise} from "./utils";
+import {log_perf, html, throttlePromise} from "./utils";
 import {DEBUG, BROWSER_MAX_HEIGHT, DOUBLE_BUFFER_RECREATE, DOUBLE_BUFFER_ROW, DOUBLE_BUFFER_COLUMN} from "./constants";
 
+import container_css from "../less/container.less";
+
```

Comment:
This is great to remove but it leaves the vestigial code - the babel plugin and symbol un `utils.js`